Chapter1/4-Shaders/Shaders/shader.vert

```diff
@@ -0,0 +1,17 @@
+#version 330 core
+
+// the position variable has attribute position 0
+layout(location = 0) in vec3 aPosition; 
+
+// This variable uses the keyword out in order to pass on the value to the 
+// next shader down in the chain, in this case the frag shader
+out vec4 vertexColor;
+
+void main(void)
+{
+	// see how we directly give a vec3 to vec4's constructor
+    gl_Position = vec4(aPosition, 1.0);
```

Comment:
tab-spacing here is weird

Reply:
Yeah don't know what that's about I looked in the file and it seems fine